### ISSUE TITLE
Canvas does not show up when used with Material-ui

### ISSUE THREAD

**Issue by mmigdiso** [projectstorm/react-diagrams#815]
I'm trying to embed react-canvas into the material-ui(5.0.0-alpha.27 but I observe the same problem with stable versions)
The CanvasWidget does not render unless I disable the `position: relative;` definition in .css-18f0176

I couldn't find a way to disable/overwrite the position style.

**Comment by Praveer-grover** [comment]
Yup, it is also coming for me as well. Found a hackish way, not sure if you could apply the style in a similar way or not depending on your DOM hierarchy:
![image](https://user-images.githubusercontent.com/5112582/112578549-b52fc500-8e1b-11eb-82eb-a747014029d8.png)

**Comment by dbflakes** [comment]
I am facing the same issue as well.
By overriding the CSS class, the diagram shows up but the lines joining the nodes are not properly aligned with the nodes.
Any suggestions?

**Comment by DrNguyen2525** [comment]
> I am facing the same issue as well.
> By overriding the CSS class, the diagram shows up but the lines joining the nodes are not properly aligned with the nodes.
> Any suggestions?

I tried to set the `position` to `absolute` and the diagram seems to work
